### PR TITLE
added Copying in Python part and exercises

### PR DESCRIPTION
**Pull request by hubschne**

### CODE REVIEW

File: Copying.ipynb [hunk 1]
```diff
@@ -0,0 +1,1063 @@
+{
```
Comment (DonSefi) [inline Copying.ipynb:1]: <p>Type: it should be  mutable, also better use find and replace, it happend on several occasions :D</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neuefische/ds-meetups/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (DonSefi) [inline Copying.ipynb:1]: <p>vice versa</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neuefische/ds-meetups/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (DonSefi) [inline Copying.ipynb:1]: <p>Maybe we can rename black box to "controls" or so? i find this line in the flow a bit distracting. Also, can we subscribt the 2 like?  $O_2$? :)--not in this comment it seems</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neuefische/ds-meetups/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (DonSefi) [inline Copying.ipynb:1]: <p>I love it, but what was the revenge for? :D</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neuefische/ds-meetups/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Comment (DonSefi) [inline Copying.ipynb:1]: <p>Add heading here like "additional examples" or "Lets play with it some more" or so to mark this part somehow? Now its just a bunch of random lines pretty much ( I actually think they can just be removed?!)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neuefische/ds-meetups/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

Comment (hubschne) [reply Copying.ipynb:1]: <p>thanks :)</p>

Comment (hubschne) [reply Copying.ipynb:1]: ups... haven't removed them :D

Comment (hubschne) [reply Copying.ipynb:1]: for killing my rainbow fish with her shark! thats totally her fault :D

**Pull request merged**